todo_app/app.py

```diff
@@ -1,15 +1,56 @@
 from flask import Flask
+from flask import render_template
 
 from todo_app.flask_config import Config
+import todo_app.data.session_items as session_items
+from flask import request, redirect
+import cgi
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
+_DEFAULT_ITEMS = [
+    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
+    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
+]
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    #return 'Hello World!'   
+    items = session_items.get_items()
+    return render_template('index.html',items=items)
 
+@app.route('/add_item', methods=['POST'])
+def add_item():
+    NewItem = request.form['title'] 
```

Comment:
Naming convention for variable and function names is snake_case.
```suggestion
    new_item = request.form['title'] 
```